content/en/blog/2023/experimental-dual-stack/index.md

```diff
@@ -0,0 +1,77 @@
+satisfied over IPv6 (we will refer to this as native IP family going forward).
+
+## Community Feedback
+
+Much of the feedback provided by the community for the original RFC was to update Envoy to better support dual-stack use cases
+internally instead of supporting this within Istio. This has led us to a [new design](https://docs.google.com/document/d/15LP2XHpQ71ODkjCVItGacPgzcn19fsVhyE7ruMGXDyU/edit?usp=sharing)
+where we have taken lessons learned as well as feedback and have applied them to fit a simplified design.
+
+## Current Work
+
+We have worked with the Envoy community to address numerous concerns which is a reason why dual-stack enablement has
+taken us a while to implement. We have implemented [matched IP Family for outbound listener](https://github.com/envoyproxy/envoy/issues/16804)
+and [supported multiple addresses per listener](https://github.com/envoyproxy/envoy/issues/11184). Alex Xu has also
+been working fervently to get long outstanding issues resolved, with the ability for Envoy to have a
+[smarter way to pick endpoints for dual-stack](https://github.com/envoyproxy/envoy/issues/21640). Some of these improvements
+to Envoy, such as the ability to [enable socket options on multiple addresses](https://github.com/envoyproxy/envoy/pull/23496),
+have landed in the upcoming Istio 1.17 release (e.g. [extra source addresses on inbound clusters](https://github.com/istio/istio/pull/41618)).
+
+The Envoy API changes the team has made can be found at their site at [Listener addresses](https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/listener/v3/listener.proto.html?highlight=additional_addresses)
+and [bind config](https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/core/v3/address.proto#config-core-v3-bindconfig).
+Making sure we can have proper support at both the downstream and upstream connection for Envoy is important for realizing
+dual-stack support.
+
+In total the team has submitted over a dozen PRs to Envoy and are working on a least a half dozen more to make Envoy adoption of
+dual stack easier for Istio.
+
+Meanwhile, on the Istio side you can track the progress in [Issue #40394](https://github.com/istio/istio/issues/40394).
+Progress has slowed down a bit lately as we continue working with Envoy on various issues, however, we are hoping to get
+this resolved before Istio 1.17 launches in February.
+
+It is our hope that for Istio 1.17 we are able to get rudimentary support for dual-stack working. Iris Ding (Intel) and
+Li Chun (Intel) are already working with the community for getting redirection of network traffic for ambient. We will
+announce what changes we have made in Istio 1.17 shortly after it is released.
+
+### Get involved
+
+While we do not have a publicly available version of Istio available for download with dual stack capabilities there are
+ample opportunities for those interesting in using, writing issues and fixing code for getting dual stack support to alpha
+feature status for Istio 1.18.
```

Comment:
```suggestion
feature status for Istio 1.17.
```

Reply:
Hi @jacob-delgado, according to the announcement of 1.17 release here: https://istio.io/latest/news/releases/1.17.x/announcing-1.17/#istio-dual-stack-support, it should be alpha feature in 1.17 release, so I suggest change it.

Reply:
Just want to make sure the 1.17 release notes are correct. As far as I can tell there is no feature status element for this work. If it's truly alpha we should have an enhancement PR for this so it is correctly represented in the feature table.

Reply:
Hi @ericvn, yes, it makes sense, I think you mean the PR for https://github.com/istio/enhancements, am I right? 
If yes, I will add it

Reply:
Hi @ericvn, there is a PR here: https://github.com/istio/enhancements/pull/141 for the feature status description. Thanks!

Reply:
I've linked to the enhancements PR.